client/packages/common/src/intl/currency/currency.ts

```diff
@@ -67,80 +67,120 @@ export const format: currency.Format = (
 };
 
-const currencyOptions = {
-  en: {
-    symbol: '$',
-    separator: ',',
-    decimal: '.',
-    precision: 2,
-    pattern: '!#',
-    negativePattern: '-!#',
-    format,
-  },
-  fr: {
-    symbol: '€',
-    separator: '.',
-    decimal: ',',
-    precision: 2,
-    pattern: '# !',
-    negativePattern: '-# !',
-    format,
-  },
-  'fr-DJ': {
-    symbol: 'DJF',
-    separator: '.',
-    decimal: ',',
-    precision: 2,
-    pattern: '# !',
-    negativePattern: '-# !',
-    format,
-  },
-  ar: {
-    symbol: 'ر.ق.',
-    separator: ',',
-    decimal: '.',
-    precision: 2,
-    pattern: '!#',
-    negativePattern: '-!#',
-    format,
-  },
-  es: {
-    symbol: '$',
-    separator: ',',
-    decimal: '.',
-    precision: 2,
-    pattern: '!#',
-    negativePattern: '-!#',
-    format,
-  },
-  tet: {
-    symbol: '$',
-    separator: ',',
-    decimal: '.',
-    precision: 2,
-    pattern: '!#',
-    negativePattern: '-!#',
-    format,
-  },
-  ru: {
-    symbol: '₽',
-    separator: '.',
-    decimal: ',',
-    precision: 2,
-    pattern: '# !',
-    negativePattern: '-# !',
-    format,
-  },
+export type Currencies =
+  | 'USD'
+  | 'EUR'
+  | 'NZD'
+  | 'DJF'
+  | 'QAR'
+  | 'RUB'
+  | 'SSP'
+  | 'PGK'
+  | 'COP';
+
+export const currencyOptions = (code?: Currencies) => {
+  switch (code) {
+    case 'USD':
+    case 'NZD':
+      return {
+        symbol: '$',
+        separator: ',',
+        decimal: '.',
+        precision: 2,
+        pattern: '!#',
+        negativePattern: '-!#',
+        format,
+      };
+    case 'EUR':
+      return {
+        symbol: '€',
+        separator: '.',
+        decimal: ',',
+        precision: 2,
+        pattern: '# !',
+        negativePattern: '-# !',
+        format,
+      };
+    case 'DJF':
+      return {
+        symbol: 'DJF',
+        separator: ',',
+        decimal: '.',
+        precision: 2,
+        pattern: '# !',
+        negativePattern: '-# !',
+        format,
+      };
+    case 'QAR':
+      return {
+        symbol: 'ر.ق.',
+        separator: ',',
+        decimal: '.',
+        precision: 2,
+        pattern: '!#',
+        negativePattern: '-!#',
+        format,
+      };
+    case 'RUB':
+      return {
+        symbol: '₽',
+        separator: '.',
+        decimal: ',',
+        precision: 2,
+        pattern: '# !',
+        negativePattern: '-# !',
+        format,
+      };
+    case 'SSP': {
+      return {
+        symbol: 'SSP',
+        pattern: '# !',
+        negativePattern: '-# !',
+        format,
+      };
+    }
+    case 'PGK': {
+      return {
+        symbol: 'K',
+        separator: '.',
+        decimal: ',',
+        precision: 2,
+        pattern: '# !',
+        negativePattern: '-# !',
+        format,
+      };
+    }
+    case 'COP': {
+      return {
+        symbol: '$',
+        pattern: '# !',
+        negativePattern: '-# !',
+        format,
+      };
+    }
+    default:
```

Comment:
How about doing this?
```suggestion
    case 'USD':
    case 'NZD':
    default:
```

and removing the first entry in the case statement?